components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -132,98 +142,114 @@ export default function ProjectSettingsView() {
                     Update Name
                 </Button>
             </form>
-            <Heading2 className="mt-12">Prebuilds</Heading2>
-            <Subheading>Choose the workspace machine type for your prebuilds.</Subheading>
-            <div className="max-w-md">
-                <SelectWorkspaceClassComponent
-                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                    onSelectionChange={setWorkspaceClassForPrebuild}
+            <div>
+                <Heading2 className="mt-12">Prebuilds</Heading2>
+                <CheckboxInputField
```

Comment:
**thought**: A checkbox might not be the best way to communicate clearly or handle properly the action here which includes also installing a webhook, but could be ok as an MVC change for now. 

I'd recommend a placeholder text as described in [EXP-589](https://linear.app/gitpod/issue/EXP-589/design-for-project-detail-page#comment-122ffba0) (internal). Then we could use a button with a proper loading state and maybe even a confirmation modal. Still fine to leave this out of the scope of this first iteration.

Reply:
I wanted to start with something simple, a binary switch seems to do the job. But in fact it is a complex action, not just flipping a state. And the tasks ahead will increase the complexity behind the scene even more, which we want to be simple in front of our users. Let's see how we can get there incrementally.

Reply:
We agreed to tackle this in separate task including design improvements.

Reply:
DEAL—Cross-posting [relevant discussion](https://gitpod.slack.com/archives/C05QPA08XAA/p1694680306699439?thread_ts=1694678643.951649&cid=C05QPA08XAA) (internal) and [relevant comment](https://linear.app/gitpod/issue/EXP-572/add-control-over-prebuilds-under-project-settings#comment-3430b5fd) (internal) for visibility. Cc. @loujaybee